### PR TITLE
Feature/cldsrv 223 implement putbuckettagging

### PR DESCRIPTION
**Pull request by ghost**
Cherry-picked the first two commits from https://github.com/scality/cloudserver/pull/4451.

Added some new tests.

**Monitoring for the API is added in https://github.com/scality/cloudserver/pull/4594**

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ models:
       env: *global-env
   - ShellCommand: &yarn-install
       name: install modules
-      command: yarn install --ignore-engines --frozen-lockfile --network-concurrency 1
+      command: yarn install --ignore-engines --frozen-lockfile --network-concurrency=1
```
Comment (XinLi0207) [inline eve/main.yml:95]: here should we use `=` or space?

Comment (ghost) [reply eve/main.yml:95]: I am guessing both are valid, but as this was changed ion 7.10, to stay consistent, I wanted to keep this change while cherry-picking

**Review by XinLi0207 — APPROVED** [review]
LGTM

**Review by miniscruff — APPROVED** [review]

**Pull request merged**